README.md

```diff
@@ -19,60 +22,69 @@ npm install --save dreamjs
 ## Usage
 
 ### Hello World
+
 ```js
 var dream = require('dreamjs');
 
 var helloworld = dream.output();
 ```
+
 The variable helloworld now contains:
 ```js
 { Dream: 'Hello World' }
 ```
 
 ### Callbacks
-Currently there is two ways you can get the output from DreamJS. By storing it in a variable or by passing a callback to it to receive the result. In the future it will also be possible to use it with Promises and Streams.
+
+Currently there is two ways you can get the output from DreamJS. By storing it in a variable or by passing a callback
```

Comment:
I will chanche it in the next commit.

Right now I am working on making the Custom Type resolution asynchornous so it will be posible things like consuming a API to generate the data that will be placed in the final json.